index.html

```diff
@@ -0,0 +1,159 @@
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/555b0c0f91.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href=".\styles.css">
+    <!-- Link to bootstrap.min.css-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css">
+    <!-- Link to jquery and popper.min.js -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <!-- Link to bootstrap.min.js -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    <title>Interior Design.</title>
+</head>
+<body>
+    <section class="main-section">
+     <header class="navbar">
+        <nav class="navbar-1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a class="nav-items-11" href="#">English</a>
+            <a class="nav-items-12" href="#">My Page</a>
+            <a class="nav-items-13" href="#">Logout</a>
+        </nav>
+        <nav class="navbar-2">
+          <a href="./index.html"><h2 class="nav-logo">SUAVE INTERIORS</h2></a>
+            <!-- <a class="nav-items" href="./index.html">Home</a> -->
+            <a class="nav-items" href="./about.html">About</a>
+            <a class="nav-items" href="#">Program</a>
+            <a class="nav-items" href="#">Join</a>
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
```

Comment:
[Optional] I recommend that you adjust the bar to match the design exactly. The light-coloured bar above the dark-coloured bar does not appear in the design guideline.

Your design:
![image](https://user-images.githubusercontent.com/20554583/191711564-26976dae-7160-4c03-898f-db715cb4844b.png)

Design guideline:
![image](https://user-images.githubusercontent.com/20554583/191711632-245d337e-30b8-4ec3-81c3-8bbe3c7ed3a1.png)